assembly/guide-configs/app-dev/features.json

```diff
@@ -4,7 +4,7 @@
   "id": "features",
   "title": "Application development - Features",
   "sidebar_label": "Features",
-  "description": "The Features section of the Temporal Application development guide provides basic implementation guidance on how to use many of the development features available to Workflows and Activities in the Temporal Platform.",
+  "description": "The Features section of the Temporal Application Developer's Guide provides basic implementation guidance on how to use many of the development features available to Workflows and Activities in the Temporal Platform.",
```

Comment:
Seems like it would be a generic "Temporal Application developer's guide"

Reply:
maybe slightly simpler without possessive? "Temporal Application developer guide"